presentation-api/controlling-ua/getAvailability.html

```diff
@@ -1,69 +1,53 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Presentation API - monitor screen availability tests for Controlling User Agent</title>
+<title>Getting the presentation displays availability information.</title>
 <link rel="author" title="Marius Wessel" href="http://www.fokus.fraunhofer.de">
-<link rel="help" href="http://w3c.github.io/presentation-api/#dfn-controlling-user-agent">
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs">
+<link rel="help" href="http://w3c.github.io/presentation-api/#dfn-presentation-display-availability">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="common.js"></script>
 
-<script>
-    // ---------------------------------
-    // Helper Function
-    // ---------------------------------
-    var createRequestObject = function () {
-        var request = new PresentationRequest(presentationUrls);
-        return request;
-    }
-
-    // ---------------------------------
-    // Screen Availability Tests - begin
-    // ---------------------------------
-
-    // Instance of Promise Test
-    test(function () {
-        var request = createRequestObject();
-        assert_true(request.getAvailability() instanceof Promise);
-    }, 'The request is an Promise.')
-
-    // Instance of PresentationRequest Test
-    test(function () {
-        var request = createRequestObject();
-        assert_true(request instanceof PresentationRequest);
-    }, 'The request is an instance of PresentationRequest.')
+<p>The test passes if a "PASS" result appears.</p>
 
-    // Instance of PresentationAvailability Test
-    promise_test(function () {
-        var request = createRequestObject();
-
-        return request.getAvailability()
-                .then(function (availability) {
-                    assert_true(availability instanceof PresentationAvailability);
-                });
-    }, "The promise is an instance of PresentationAvailability");
-
-    // Availability.value is set Test
-    promise_test(function () {
-        var request = createRequestObject();
+<script>
 
-        return request.getAvailability()
-                .then(function (availability) {
-                    assert_true(typeof availability.value == 'boolean');
-                });
-    }, "The availability has an boolean value.");
+    // ---------------------------------------
+    // Presentation Availability Tests - begin
+    // ---------------------------------------
 
+    promise_test(function(t) {
+        var availability;
 
-    // Best Case Scenario Test
-    // -----------------------
-    promise_test(function () {
-        var request = createRequestObject();
+        // instance of PresentationRequest
```

Comment:
I'm fan of comments in the code, but I note all the "assert" comments already appear in the assert calls themselves, so I think you can simply drop them throughout. That's just a suggestion, of course :)